src/amuse/test/suite/core_tests/test_pickle.py

```diff
@@ -1,37 +1,30 @@
-from amuse.test import amusetest
-
+import subprocess
 import pickle
+import sys
+import os
 
-from amuse.support.exceptions import AmuseException
+from amuse.test import amusetest
 
-from amuse.units import core
 from amuse.units import si
 from amuse.units import nbody_system
-from amuse.units import generic_unit_system
 from amuse.units.quantities import zero
-from amuse.units.units import *
-from amuse.units.constants import *
+from amuse.units.units import m, km, kg, parsec, stellar_type
 
 from amuse.datamodel import Particles, parameters
 
-import subprocess
-import pickle
-import sys
-import os
-
 
 class TestPicklingOfUnitsAndQuantities(amusetest.TestCase):
 
     def test1(self):
-        km = 1000 * m
-        self.assertEqual(1000, km.value_in(m))
-        pickled_km = pickle.dumps(km)
-        unpickled_km = pickle.loads(pickled_km)
-        self.assertEqual(1000, unpickled_km.value_in(m))
+        kilometer = 1000 * m
```

Comment:
renaming it here since 'km' is already defined... could probably just as easily use the pre-defined 'km' unit, but maybe there was a reason not to?